scripts/learninglabauto.py

```diff
@@ -1,7 +1,7 @@
 import os
 import re
 
-subdir = ""
+subdir = "Slack-Apps"
```

Comment:
I think it would be better to change the learninglabauto.py file after the PR is merged (so @lazyplatypus would it be ok to make an exception to commit directly to the repo without a PR in this case?)

Reply:
Yes this is fine

Reply:
got it